doc/tutorials/transformer/te_init.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>"If we need to use <strong>a</strong> Huggingface token let's put it into an <strong>PyExasol</strong> connection object."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Well spotted typo. This is actually a connection object in the DB, not pyexasol.</p>

Reply:
<p>please fix</p>